source/templates/displaying-a-list-of-items.md

```diff
@@ -23,13 +23,21 @@ The above example will print a list like this:
 
 Like everything in Handlebars, the `{{#each}}` helper is bindings-aware.
 If your application adds a new item to the array, or removes an item,
-the DOM will be updated without having to write any code.
+the DOM will be updated without having to write any code. Because
+Glimmer (Ember's rendering engine) will try to re-use DOM whenever
+possible, it needs a way to uniquely identify each loop item in order
+to prevent unexpected results. This is done by passing the `key`
+attribute to the `{{#each}}` helper. The value of the `key` attribute
+will be the property Ember looks up on the loop object, in this case
```

Comment:
Seems good to me, also @tomdale cleaned up what I had originally written to be good :dancer: